.github/workflows/test-integration-unreleased.yml

```diff
@@ -66,6 +66,8 @@ jobs:
       - name: Integration test
         working-directory: go-client
         run: make test-integ race=true
+        env:
+          OPENSEARCH_GO_SKIP_JSON_COMPARE: true
```

Comment:
For the reader it's totally unclear why this is done. Is there a cleaner version that can be written into the test? Maybe something like checking whether we're running against a -SNAPSHOT version of OpenSearch from info()?

Reply:
I understand your concern. To explain my self. I tried doing a simple way of skipping all the validation checks without touching each test again after this issue came up yesterday.

Yes, maybe doing the info call after the client init in each test and putting the validation call behind an if checking the version could be a solution.